kind/config.yaml

```diff
@@ -2,16 +2,10 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
 nodes:
 - role: control-plane
-  kubeadmConfigPatches:
-  - |
-    kind: InitConfiguration
-    nodeRegistration:
-      kubeletExtraArgs:
-        node-labels: "ingress-ready=true"
   extraMounts:
-  - containerPath: /var/lib/kubelet/config.json
-    # this is the path in gh action where the config is saved
-    hostPath: /tmp/config.json
+    - containerPath: /var/lib/kubelet/config.json
+      # this is the path in gh action where the config is saved
+      hostPath: /tmp/config.json
   extraPortMappings:
```

Comment:
below, can you change hostPort: 80 to `hostPort: 8888`  and below that, hostPort: 443 to `hostPort: 4433` now that we are running non-root (and thanks for making it non-root!)

Reply:
or maybe we don't need the extraPortMappings at all? testing...

Reply:
confirmed, all is good without the extraPortMappings